src/main/java/com/bina/varsim/types/variant/alt/Alt.java

```diff
@@ -0,0 +1,118 @@
+package com.bina.varsim.types.variant.alt;
+
+import com.bina.varsim.types.ChrString;
+import com.bina.varsim.types.FlexSeq;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by guoy28 on 11/18/16.
+ * class for ALT field in VCF format
+ */
+public class Alt {
+  private SymbolicAllele symbolicAllele;
+  private Breakend breakend;
+  private FlexSeq seq;
+  public Alt() {}
+
+  public SymbolicAllele getSymbolicAllele() {
+    return symbolicAllele;
+  }
+
+  public void setSymbolicAllele(SymbolicAllele symbolicAllele) {
+    this.symbolicAllele = symbolicAllele;
+  }
+
+  public Breakend getBreakend() {
+    return breakend;
+  }
+
+  public void setBreakend(Breakend breakend) {
+    this.breakend = breakend;
+  }
+
+  public FlexSeq getSeq() {
+    return seq;
+  }
+
+  public void setSeq(FlexSeq seq) {
+    this.seq = seq;
+  }
+
+  public enum SVType {
+    DEL, INS, DUP, INV, CNV,
+    BND; //breakend
+    public enum SVSubtype {
+      TANDEM, //consecutive events
+      ME, //mobile element
+      TRA, //translocation (cut-and-paste)
+      IPS; //interspersed
+    }
+  }
+  public static class SymbolicAllele{
+    /**
+     * looks like <DUP:TANDEM>
+     */
+    private SVType major;
+    private SVType.SVSubtype minor;
+
+    @Override
+    public String toString() {
+      return "<" + major + (minor == null? ">" : (":" + minor + ">"));
+    }
+
+    public SymbolicAllele(String major, String minor) {
+      this.major = SVType.valueOf(major);
+      this.minor = SVType.SVSubtype.valueOf(minor);
+    }
+    public SymbolicAllele(String major) {
+      this.major = SVType.valueOf(major);
+      this.minor = null;
+    }
+
+    /**
+     * parse <X:Y>
+     * @param alt
+     */
+    public static SymbolicAllele SymbolicAlleleFactory(String alt) {
+      Pattern r = Pattern.compile("<([^<:>]+):?([^<:>]*)>");
```

Comment:
This could be a final variable outside of the function so that this pattern does not need to be compiled every function call.